src/main/java/com/iexec/worker/pubsub/SubscriptionService.java

```diff
@@ -184,7 +184,7 @@ public void handleTaskNotification(TaskNotification notif) {
                 case PLEASE_CONTRIBUTE:
```

Comment:
```suggestion
                //TODO Add PLEASE_COMPUTE etc
                case PLEASE_CONTRIBUTE:
```

Reply:
I don't think this is a good idea because the worker is the only one who knows when to compute (when the result is/is not found.